### PR TITLE
Update LIT6174Severus318.xml

### PR DESCRIPTION
**Pull request by eu-genia**
https://github.com/BetaMasaheft/Documentation/issues/1866

### CODE REVIEW

**Comment by eu-genia** [comment]
Ok the Severos should be clean now

**Comment by eu-genia** [comment]
please approve if all OK )

File: 1001-2000/LIT1299Dersan.xml [hunk 1]
```diff
@@ -67,13 +67,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
         <div type="bibliography">
         <listBibl type="secondary">
```
Comment (MarcinKrawczuk) [inline 1001-2000/LIT1299Dersan.xml:69]: Maybe you could add bm:Tedros2017Sawiros  to the bibliography as edition?

**Review by MarcinKrawczuk — APPROVED** [review]
BTW I have a long list of possible doublets for future...

**Pull request merged**